.github/workflows/nix.yml

```diff
@@ -42,5 +42,24 @@ jobs:
         echo Building the project and its devShell
         nix build .#check --log-lines 500 --show-trace
 
+        echo Build the bundle
+        nix build .#pact-binary-bundle --log-lines 500 --show-trace --out-link pact-binary-bundle
+        tar -zcvf pact-binary-bundle.${{ matrix.os }}.tar.gz $(readlink pact-binary-bundle)
```

Comment:
In retrospect, we should probably decide on what this should be named and come up with a convention for it. `pact-binary-bundle.*.tar.gz` isn't very friendly or informative